boot/efi/multiboot.c

```diff
@@ -295,10 +296,38 @@ MbiSetPointerToAcpiTable(void)
     *       bootloader under QEMU: that's slower for tests and limiting for
     *       debugging purposes.
     */
-   gMbi->apm_table = (u32)tablePaddr;
+   // gMbi->apm_table = (u32)tablePaddr;
+   
+   extra_boot_info->RSDP = (u32)tablePaddr;
```

Comment:
That's good, but the whole comment above it doesn't apply here anymore. It should be moved to the place where `gMbi->apm_table` is set, because there is were we're going the "hack".